modules/ROOT/pages/_partials/include-alerts-config.adoc

```diff
@@ -53,16 +53,16 @@ TODO _FUTURE?
 [[notifications]]
 === Notifications
 
-Configure a notification to email when an alert is triggered.
+Configure a notification to email when an advanced alert is triggered.
```

Comment:
email > send? send by email? Or just "configure a notification for when..."?

Reply:
```suggestion
Configure a notification to send an email when an advanced alert is triggered
```